events/events.go

```diff
@@ -47,14 +49,15 @@ type (
 	}
 
 	EventContractSetUpdate struct {
-		Name      string                 `json:"name"`
-		Contracts []api.ContractMetadata `json:"contracts"`
-		Timestamp time.Time              `json:"timestamp"`
+		Name        string                 `json:"name"`
+		ContractIDs []types.FileContractID `json:"contractIDs"`
```

Comment:
I'm going back and fourth here on whether to pass the metadata or the contract ids. It's a little awkward to pass the metadata because we'd have to fetch it from the database using a background context in the `bus` handler... we updated the contract set using the ids so it makes sense to only pass those I think.